portfolio/pom.xml

```diff
@@ -41,4 +55,7 @@
       </plugin>
     </plugins>
   </build>
-</project>
\ No newline at end of file
+</project>
```

Comment:
What do you think about removing this extra whitespace?

Reply:
Done!